docs/Streams.md

```diff
@@ -170,6 +170,17 @@ The application regains full ownership of a buffer after it get a receive notifi
 If the application accepts all the buffer's bytes **inline** from the receive notification, by returning `QUIC_STATUS_SUCCESS` and setting `TotalBufferLength` appropriately,
 it can free or reuse the buffer while in the notification handler.
 
+If more data is received on the stream than buffer space was provided by the application,
+MsQuic will emit a `QUIC_STREAM_EVENT_INSUFFICIENT_RECEIVE_BUFFER` notification.
```

Comment:
```suggestion
If more data is received on the stream than buffer space was provided by the application, MsQuic will emit a `QUIC_STREAM_EVENT_INSUFFICIENT_RECEIVE_BUFFER` notification.
```
nit: for docs, in general we put each sentence on a single line. No manual line wraps.

Reply:
Fixed. I dislike long lines in md documents (hard to diff and review), but I guess I'll just do small sentences :)